### PR TITLE
docs: move to v1.0.0

### PR DESCRIPTION
**Pull request by kpouget**

### CODE REVIEW

**Comment by kpouget** [comment]
@ArangoGutierrez can we merge this?
this branch (`openshift-psap:branch-v1.0`) will be tagged as `v1.0.0` after this merge \o/

**Comment by kpouget** [comment]
@ArangoGutierrez can you fix the EOF EOL?

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,11 +2,8 @@
 
 (Organized release by release)
 
-Changes since version May 7th, 2021
------------------------------------
-
-Toolbox
-~~~~~~~
+Changes in version 1.0
```
Comment (ArangoGutierrez) [inline CHANGELOG.rst:5]: there are no "changes" since this is the first version

File: docs/conf.py [hunk 2]
```diff
@@ -21,23 +21,12 @@
...
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-import datetime
-version = datetime.date.today().strftime('%d, %b %Y')
-# The full version, including alpha/beta/rc tags.
-release = 'git-{}/{}'.format(git_branch, git_short_hash)
+release = 'version 1.0.0'
```
Comment (ArangoGutierrez) [inline docs/conf.py:29]: `v1.0.0` is better

**Review by ArangoGutierrez — CHANGES_REQUESTED** [review]

File: docs/conf.py (hunk 2 above)
Comment (kpouget) [reply docs/conf.py]: fixed

File: CHANGELOG.rst (hunk 1 above)
Comment (kpouget) [reply CHANGELOG.rst:5]: yes, I wasn't sure how to phrase it, it's the first "named release", but the changelog is not empty...?

Comment (ArangoGutierrez) [reply CHANGELOG.rst:5]: given that this is release 1, why not going for `features` or `functionality` that way, change logs make more sense moving forward, bc we will be `changing` the initial functionality/feature set

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Pull request merged**